src/api/libcellml/units.h

```diff
@@ -492,19 +492,20 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     UnitsPtr clone() const;
 
+private:
+    Units(); /**< Constructor */
+    explicit Units(const std::string &name); /**< Constructor with std::string parameter*/
+
     /**
-     * @brief Set the import source of these units.
+     * @brief Set the import source of this units.
      *
+     * Virtual method implementing ImportedEntity::setImportSource, @private.
      * If these units are already located in a Model instance, then the
      * import source is added to the Model too.
      *
-     * @param importSource The @c ImportSourcePtr to add to this @c Units item.
+     * @param importSource The @c ImportSourcePtr to add to this @ref Units.
      */
-    void setImportSource(ImportSourcePtr &importSource);
-
-private:
-    Units(); /**< Constructor */
-    explicit Units(const std::string &name); /**< Constructor with std::string parameter*/
+    void doSetImportSource(const ImportSourcePtr &importSource) override;
```

Comment:
Just as a reminder, `doSetImportSource()` will have to be removed from the API at some point. It really has nothing to do here.